engines/testbed/encoding.cpp

```diff
@@ -20,14 +20,13 @@
  *
  */
 
-#include "common/encoding.h"
 #include "common/system.h"
 #include "engines/testbed/encoding.h"
 
 namespace Testbed {
 
 TestExitStatus Encodingtests::testConversionUnicodeMachineEndian() {
-	Testsuite::displayMessage("Encoding conversion tests.\nTo test iconv conversion, compile with --enable-iconv.\nTo test backend conversion, compile with --disable-iconv.\nSome tests may fail without iconv, because some backends support only a handful of conversions.");
```

Comment:
It would be a good idea to move this entirely into the unit test suite, since this no longer relies on the backend being initialized.